eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -54,6 +90,10 @@ function Get-javascript-DocsMsTocChildrenForManagementPackages($packageMetadata,
   return @($packageMetadata.Package)
 }
 
+function Get-javascrpt-RepositoryLink ($packageInfo) {
```

Comment:
Is there a part of testing that could have caught this misspelling?

```suggestion
function Get-javascript-RepositoryLink ($packageInfo) {
```

Reply:
There is a default value for this. That's why it didn't fail.